crypto/srp/srp_vfy.c

```diff
@@ -409,6 +409,11 @@ int SRP_VBASE_init(SRP_VBASE *vb, char *verifier_file)
 
     error_code = SRP_ERR_OPEN_FILE;
 
+    if (verifier_file == NULL) {
+        ERR_raise(ERR_LIB_X509, ERR_R_BIO_LIB);
```

Comment:
ERR_R_BIO_LIB is wrong here. ERR_R_PASSED_NULL_PARAMETER is the right reason code.